### DISCUSSION TITLE
How to hide/delete waifu AI tab?

### DISCUSSION THREAD

**Discussion by Chiron8** [end-4/dots-hyprland#845] [Q&A] [answered]
Hi, I was wondering how you can either hide or delete the AI waifu tab. I can't find the files for it on my system (I'm using arch linux) Thanks!

**Comment by end-4** [comment, marked as answer]
hi,
there's no "AI waifu" tab, but there are 2 AI tabs and 2 waifu tabs. 
regardless, to hide pages, add the following to `~/.config/ags/user_options.js` in the `userConfigOptions` object then remove the stuff you don't want:
```js
    'sidebar': {
        'pages': {
            'order': ["apis", "tools"],
            'apis': {
                'order': ["gemini", "gpt", "waifu", "booru"], // Remove the pages you don't want on this line
            }
        }
    }
```

**Comment by Chiron8** [comment]
Thank you so much!
